src/app/pages/products-list/products-list.component.html

```diff
@@ -24,37 +24,64 @@
     <app-card class="card" [product]="productsFromNgIf[4]" (buy)="onProductBuy($event)"></app-card> -->
 
     <!-- Раскомментрировать для ДЗ -->
-    <!-- <ng-container
+    <ng-container
         *appPagination="
             let productsGroup of products;
+            let appPaginationOf = appPaginationOf;
```

Comment:
Почему так, а не через `as`?
```suggestion
            let productsGroup of products as appPaginationOf;
```

Reply:
это вообще не нужно уже, убрала)